src/docker.test.ts

```diff
@@ -87,14 +87,15 @@ describe("Docker images", (): void => {
   const assertSaveDockerImages = (
     cacheHit: boolean,
     readOnly = false,
+    prevSave = false,
   ): void => {
     expect(core.getInput).nthCalledWith<[string, InputOptions]>(1, "key", {
       required: true,
     });
     expect(core.getState).nthCalledWith<[string]>(1, docker.CACHE_HIT);
     if (!cacheHit) {
       expect(core.getInput).lastCalledWith("read-only");
```

Comment:
We also want to assert that `cache.restoreCache` was called with the expected arguments if `!readOnly`.